src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/handlers/presubmit/ChangeDirectionDueDateNotificationHandler.java

```diff
@@ -0,0 +1,94 @@
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ccd.callback.PreSubmitCallbackStage;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.handlers.PreSubmitCallbackHandler;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.service.HomeOfficeInstructService;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.service.NotificationsHelper;
+
+@Slf4j
+@Component
+public class ChangeDirectionDueDateNotificationHandler implements PreSubmitCallbackHandler<AsylumCase> {
+
+    private HomeOfficeInstructService homeOfficeInstructService;
+    private NotificationsHelper notificationsHelper;
+
+    public ChangeDirectionDueDateNotificationHandler(
+        HomeOfficeInstructService homeOfficeInstructService,
+        NotificationsHelper notificationsHelper
+    ) {
+        this.homeOfficeInstructService = homeOfficeInstructService;
+        this.notificationsHelper = notificationsHelper;
+    }
+
+    @Override
+    public boolean canHandle(PreSubmitCallbackStage callbackStage, Callback<AsylumCase> callback) {
+        requireNonNull(callbackStage, "callbackStage must not be null");
+        requireNonNull(callback, "callback must not be null");
+
+        return callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
+               && callback.getEvent() == Event.CHANGE_DIRECTION_DUE_DATE;
+    }
+
+    @Override
+    public PreSubmitCallbackResponse<AsylumCase> handle(
+        PreSubmitCallbackStage callbackStage, Callback<AsylumCase> callback) {
+
+        if (!canHandle(callbackStage, callback)) {
+            throw new IllegalStateException("Cannot handle callback");
+        }
+        log.info("Preparing to send {} notification to HomeOffice for event {}",
+            DEFAULT.toString(), callback.getEvent());
+
+        AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
+
+        final String homeOfficeReferenceNumber = notificationsHelper.getHomeOfficeReference(asylumCase);
+
+        final String caseId = notificationsHelper.getCaseId(asylumCase);
+
+        final HomeOfficeInstruct homeOfficeInstruct =
+            new HomeOfficeInstruct(
+                notificationsHelper.getConsumerReference(caseId),
+                homeOfficeReferenceNumber,
+                notificationsHelper.getMessageHeader(),
+                DEFAULT.name(),
+                getNote(asylumCase)
+            );
+
+        log.info("Finished constructing {} notification request for caseId: {}, HomeOffice reference: {}",
+            DEFAULT.toString(), caseId, homeOfficeReferenceNumber);
+
+        final String notificationStatus = homeOfficeInstructService.sendNotification(homeOfficeInstruct);
+
+        asylumCase.write(HOME_OFFICE_CHANGE_DIRECTION_DUE_DATE_INSTRUCT_STATUS, notificationStatus);
```

Comment:
Do we not need to differentiate between evidence bundle and request review changes?

Reply:
added new status